### ISSUE TITLE
Abs() not working correctly with variables 

### ISSUE THREAD

**Issue by cristianbuse** [WaynePhillipsEA/twinbasic#180]
**Describe the bug**
While re-dimensioning an array, tB does not allow bounds containing method calls with local variables as argument

**To Reproduce**
```VBA
Module Test
    Sub Main()
        Dim arr() As Variant
        Dim upperBound As Long = 4
        
        ReDim arr(0 to 5)                   'Works fine!
        ReDim arr(0 To Abs(5))              'Works fine!
        ReDim arr(0 To 1 + Abs(4))          'Works fine!
        ReDim arr(0 To Abs(upperBound))     'Runtime error!
        ReDim arr(0 To 1 + Abs(upperBound)) 'Runtime error!
    End Sub
End Module
```

**Expected behavior**
In VBA the above ```ReDim``` statements are valid.

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise V1909
 - twinBASIC compiler v0.10.871

**Comment by WaynePhillipsEA** [comment]
This was not a problem with the ReDim implementation, but a more general problem with the Abs() implementation.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.933, thanks for reporting.
